### DISCUSSION TITLE
EKS Ingress

### DISCUSSION THREAD

**Discussion by dubb-b** [emqx/emqx#4689] [General]
Just curious if you have any examples of having the ingress for the api and the dashboard running in EKS/AWS ALB/ELB/Classic?

Do you recommend using Classic still or can an ALB be used?

Thanks in advance.

**Comment by Rory-Z** [comment]
Hi, @dubb-b 
Sorry, we don’t have any examples of this
Maybe you can try it, we welcome you to contribute articles or examples to us
